GraknClient.java

```diff
@@ -19,169 +19,25 @@
 
 package grakn.client;
 
-import grakn.client.common.exception.GraknClientException;
-import grakn.client.rpc.Address;
-import grakn.client.rpc.RPCDatabaseManager;
-import grakn.client.rpc.RPCSession;
-import grakn.common.collection.Pair;
-import grakn.protocol.cluster.ClusterProto;
-import grakn.protocol.cluster.GraknClusterGrpc;
-import io.grpc.Channel;
-import io.grpc.ManagedChannel;
-import io.grpc.ManagedChannelBuilder;
-import io.grpc.StatusRuntimeException;
-import org.slf4j.Logger;
-import org.slf4j.LoggerFactory;
+import grakn.client.rpc.RPCClient;
+import grakn.client.rpc.cluster.ClusterClient;
 
-import java.util.Map;
-import java.util.Set;
-import java.util.concurrent.TimeUnit;
-import java.util.stream.Collectors;
+public interface GraknClient {
+    String DEFAULT_ADDRESS = "localhost:1729";
 
-import static grakn.client.common.exception.ErrorMessage.Client.CLUSTER_UNABLE_TO_CONNECT;
-import static grakn.common.collection.Collections.pair;
-
-public class GraknClient {
-    public static final String DEFAULT_ADDRESS = "localhost:1729";
-
-    public static Core core() {
+    static Grakn.Client core() {
```

Comment:
Note that because this is now an interface, `public` becomes implicit on these methods